lib/semantics/check-do.cc

```diff
@@ -52,6 +52,20 @@ static void SayWithDo(SemanticsContext &context, parser::CharBlock stmtLocation,
   context.Say(stmtLocation, message).Attach(doLocation, GetEnclosingDoMsg());
 }
 
+static void SayWithDeclaration(SemanticsContext &context,
+    const parser::CharBlock &location, const parser::MessageFixedText &&message,
+    const parser::CharBlock &name, const Symbol &symbol) {
+  if (const auto *details{symbol.detailsIf<UseDetails>()}) {
+    context.Say(location, message, name)
```

Comment:
`message` should not be `const` and the uses should be `std::move(message)`.

Reply:
Thanks, Tim.  I'll make that change.